conf/client.properties

```diff
@@ -1,31 +1,31 @@
 # Optional. A hostname of the Server. Use localhost by default if not specified.
-#scalar.ledger.client.server_host=localhost
+scalar.ledger.client.server_host=sandbox.scalar-labs.com
 
-# Optional. A port number of the Server. Use 50051 by default if not specified.
-#scalar.ledger.client.server_port=50051
+# Optional. A port number of the Server.
+scalar.ledger.client.server_port=443
 
 # Required. An ID of a certificate holder.
 # It must be configured for each private key and unique in the system.
-scalar.ledger.client.cert_holder_id=foo
+scalar.ledger.client.cert_holder_id=please_change_me
```

Comment:
Maybe we should generate this on the registration server.  @feeblefakie what do you think?